pkg/controller/cassandracluster/generator.go

```diff
@@ -577,33 +585,15 @@ func createEnvVarForCassandraContainer(cc *api.CassandraCluster, status *api.Cas
 				},
 			},
 		},
-		v1.EnvVar{
-			Name: "POD_NAME",
```

Comment:
Did you remove this one because it's unused ?

Reply:
Correct